integration/setup/setup.go

```diff
@@ -342,3 +339,51 @@ func insertBenchmarkProvider(tb testing.TB, ctx context.Context, collection *mon
 
 	return
 }
+
+// cleanupCollection deletes all documents in the collection for ferretdb-yugabytedb backend
+// or drops the collection for other backends. The drop collection error is optionally ignored.
```

Comment:
Why do we need this boolean flag? What error could be returned?

Reply:
Removed the flag, thanks :pray: